packages/orchestrator-ui-components/src/components/WFOSteps/WFOStepList/WFOStepList.tsx

```diff
@@ -11,6 +11,33 @@ export interface WFOStepListProps {
     startedAt: string;
 }
 
+export const stateDelta = (prev: State, curr: State) => {
```

Comment:
Matter of taste but I'm more for complete words instead of abbrv. So previous instead of prev

Reply:
Maybe even previousState (or if agree with my type name suggestion in a different comment previousStepState)